2022/anthony_nocentino/WindowsContainers/demo/demo1/v3/mywebapp/obj/Debug/netcoreapp3.1/webapp.GeneratedMSBuildEditorConfig.editorconfig

```diff
@@ -1,3 +1,3 @@
 is_global = true
 build_property.RootNamespace = webapp
-build_property.ProjectDir = C:\Users\aen\source\repos\WebApplicationNetCore\
+build_property.ProjectDir = c:\summit-materials\2022\anthony_nocentino\WindowsContainers\demo\demo1\v3\mywebapp\

```

Comment:
This part intentional?  If it's good with @nocentino  I can merge!

Reply:
Nope, that debug directory can be deleted. I'll submit an updated PR.